apps/codeforafrica/src/lib/data/blockify/posts.js

```diff
@@ -0,0 +1,29 @@
+import {
+  getPosts,
+  formatPost,
+  getTagsByPrimaryTag,
+} from "@/codeforafrica/lib/data/utils/posts";
+
+async function posts(block, api, context) {
+  const { primaryTag, labels, stories = {} } = block;
+  const { query } = context;
+  const { featured, title } = stories;
+  const featuredStory = featured && (formatPost(featured, primaryTag) || null);
```

Comment:
```suggestion
  const { featured: featuredStory, title } = stories;
  const featured = featuredStory ? formatPost(featuredStory, primaryTag) : null;
```

`formatPost` already returns null, no?